src/javaRosa.js

```diff
@@ -44,7 +44,8 @@ define([
             'default', 'short', 'long', 'audio', 'video', 'image'
         ],
         _nextItextItemKey = 1,
-        NO_MARKDOWN_MUGS = ['Item', 'Group', 'FieldList', 'Repeat'];
+        NO_MARKDOWN_MUGS = ['Item', 'Group', 'FieldList', 'Repeat'],
+        EXPERIMENTAL_UI;
```

Comment:
ok, sounds good. but only if we can name it richinput instead of richsingleline :)